sql-statements/sql-statement-create-table.md

```diff
@@ -225,8 +225,6 @@ mysql> DESC t1;
 * The `COMMENT` attribute does not support the `WITH PARSER` option.
```

Comment:
Also need to update ebnf, see https://github.com/pingcap/docs-cn/pull/11974/files#diff-05d7fa31873281b59e53a89cec973d1c909ee9fe91fde46ce677bb4700cd4cd7R62